luzer/luzer.c

```diff
@@ -219,6 +222,28 @@ TestOneInput(const uint8_t* data, size_t size) {
 
 	lua_State *L = get_global_lua_state();
 
+	char *buf = malloc(size + 1 * sizeof(*buf));
+	memcpy(buf, data, size);
+	buf[size] = '\0';
+	lua_pushlstring(L, buf, size);
+
+#ifdef LUAJIT_FRIENDLY_MODE
+	/*
+	 * TODO: Get a current mode with `jit.status()` and skip below
+	 * if compiler was disabled `jit.off()`.
+	 */
+	int res = luaJIT_setmode(L, 0, LUAJIT_MODE_ON);
```

Comment:
It looks like we don't need this `res` variable anymore, actually.
I suppose we may do something like:
```c
    if (!luaJIT_setmode(L, 0, LUAJIT_MODE_ON))
        luaL_error(L, "cannot turn a JIT compiler on");
```

Side note: I also dropped here unnecessary `{}` for the call.

Reply:
Ok, updated:

```diff
--- a/luzer/luzer.c
+++ b/luzer/luzer.c
@@ -232,16 +232,11 @@ TestOneInput(const uint8_t* data, size_t size) {
         * TODO: Get a current mode with `jit.status()` and skip below
         * if compiler was disabled `jit.off()`.
         */
-       int res = luaJIT_setmode(L, 0, LUAJIT_MODE_ON);
-       if (!res) {
+       if (!luaJIT_setmode(L, 0, LUAJIT_MODE_ON))
                luaL_error(L, "cannot turn a JIT compiler on");
-       }
-       /* Returned value is not handled. */
        luaL_test_one_input(L);
-       res = luaJIT_setmode(L, 0, LUAJIT_MODE_OFF);
-       if (!res) {
+       if (!luaJIT_setmode(L, 0, LUAJIT_MODE_OFF))
                luaL_error(L, "cannot turn a JIT compiler off");
-       }
```

Reply:
It looks like you accidentally removed the comment 
```c
/* Returned value is not handled. */
```
related to the `luaL_test_one_input(L)` call.

Reply:
My bad, thanks!